### ISSUE TITLE
`2:length(x)` wrongly detected as `1:length(x)`

### ISSUE THREAD

**Issue by etiennebacher** [etiennebacher/flint#24]
Seen in `lintr`

``` r
flint::lint_text("2L:length(parsed_expr)")
#> Original code: 2L:length(parsed_expr) 
#> Suggestion: 1:length(...) is likely to be wrong in the empty edge case. Use seq_along(...) instead.
#> 
```
